xdsl/utils/exceptions.py

```diff
@@ -47,15 +47,27 @@ class PassFailedException(DiagnosticException):
 
 
 class PyRDLError(Exception):
-    pass
+    """
+    An error in our Python to IRDL conversion.
```

Comment:
Maybe "in our Python IRDL eDSL" instead?
Python to IRDL might refer to something else